File: Exercice 1/exercice1.js
for (let num = 1; num <= 100; num++) {
    if (num % 3 === 0) {
        process.stdout.write("Hello, ");
    } else if (num % 5 === 0) {
        process.stdout.write("World, ");
    } else if (num % 7 === 0) {
        process.stdout.write("Yoo, ");
    } else {
        process.stdout.write(num + ", ");
    }
}

File: Exercice 1/exercice1.py
for num in range(1, 101):
    if num % 3 == 0:
        print("Hello", end=", ")
    elif num % 5 == 0:
        print("World", end=", ")
    elif num % 7 == 0:
        print("Yoo", end=", ")
    else:
        print(num, end=", ")
